velox/dwio/common/FormatData.h

```diff
@@ -95,7 +96,11 @@ class FormatData {
   virtual std::vector<uint32_t> filterRowGroups(
       const velox::common::ScanSpec& scanSpec,
       uint64_t rowsPerRowGroup,
-      const StatsContext& context) = 0;
+      const StatsContext& writerContext) = 0;
+
+  virtual bool rowGroupMatches(
```

Comment:
Can you add a comment for this function?

Reply:
Added this
```
  /// Test if the 'i'th RowGroup can potentially match the 'filter' using the
  /// RowGroup's statistics on all columns. Returns true if all columns in this
  /// RowGroup matches the filter, false otherwise. A column is said to match
  /// the filter if its ColumnStatistics passes the filter, or the filter for
  /// that column is NULL.
```